### PR TITLE
PPC64LE image tags should have prefix

### PR DESCRIPTION
**Pull request by MarcinGinszt**
Otherwise they overwrite our, existing images.
ISV-1495

### CODE REVIEW

**Comment by MarcinGinszt** [comment]
cc: @mayurwaghmode

File: .github/workflows/ppc64le-image.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ jobs:
         uses: redhat-actions/buildah-build@v2
         with:
           image: operator-pipelines-images
-          tags: ppc64le ${{ github.sha }}
+          tags: ppc64le-${{ github.sha }}
```
Comment (Allda) [inline .github/workflows/ppc64le-image.yml:24]: This change has the side effect of removing a floating tag from ppc64le images. Could you keep ppc64le tag and add a separate one `ppc64le-sha`?

Comment (MarcinGinszt) [reply .github/workflows/ppc64le-image.yml:24]: Floating tag is pointless, as long as the ppc64le image is built on every commit. WDYT?

Comment (MarcinGinszt) [reply .github/workflows/ppc64le-image.yml:24]: @mayurwaghmode what should we do with this tag?

Comment (mayurwaghmode) [reply .github/workflows/ppc64le-image.yml:24]: @MarcinGinszt Can we have a tag as ppc64le:latest?

Comment (MarcinGinszt) [reply .github/workflows/ppc64le-image.yml:24]: Yes, we can. But would you like to build it for every commit on every branch, or maybe only for master?

Comment (rpsene) [reply .github/workflows/ppc64le-image.yml:24]: what if we set the ppc64le-$BRANCH-${{ github.sha }} for every branch and ppc64le:$RELEASE for every new release?

Comment (MarcinGinszt) [reply .github/workflows/ppc64le-image.yml:24]: @rpsene @mayurwaghmode  You are encouraged to submit the PR in order to create a custom image tagging system. I just want to unblock us by removing the bug, that overrides images necessary for prod.

Comment (MarcinGinszt) [reply .github/workflows/ppc64le-image.yml:24]: @rpsene @mayurwaghmode  can we merge this one?

Comment (rpsene) [reply .github/workflows/ppc64le-image.yml:24]: @MarcinGinszt for now yes. We will update it later.

**Review by Allda — APPROVED** [review]

**Pull request merged**